examples/java/.gitignore

```diff
@@ -0,0 +1,13 @@
+
+build/
+
+.gradle/
+gradle/
+gradlew
+gradlew.bat
+
+**/.idea/**/workspace.xml
```

Comment:
This is a slightly odd mix of styles. `build/` and `**/.idea/...` will both have the effect of matching those files in any dir in `examples/java`. I'm fine with it either way, but have a slight preference for having them match.